app/helpers/session_header_helper.rb

```diff
@@ -1,10 +1,15 @@
+# frozen_string_literal: true
+
 module SessionHeaderHelper
-  def to_account_session(access_token, refresh_token)
+  LOWEST_LEVEL_OF_AUTHENTICATION = "level0"
+
+  def to_account_session(access_token:, refresh_token:, level_of_authentication:)
     SessionEncryptor
       .new(session_signing_key: Rails.application.secrets.session_signing_key)
       .encrypt_session(
         access_token: access_token,
         refresh_token: refresh_token,
+        level_of_authentication: level_of_authentication || LOWEST_LEVEL_OF_AUTHENTICATION,
```

Comment:
Calling `to_account_session` without `level_of_authentication` would raise an ArgumentError so `LOWEST_LEVEL_OF_AUTHENTICATION` wouldn't be used. It could be moved to the method definition:
```ruby
def to_account_session(access_token:, refresh_token:, level_of_authentication: LOWEST_LEVEL_OF_AUTHENTICATION)
```

```ruby
[10] pry(main)> def bla(a:, b:)
[10] pry(main)*   { foo: a, bar: b || "default" }
[10] pry(main)* end
=> :bla
[11] pry(main)> bla(a: "foo")
ArgumentError: missing keyword: :b
from (pry):16:in `bla'
[12] pry(main)> def bla(a:, b: "default")
[12] pry(main)*   { foo: a, bar: b }
[12] pry(main)* end
=> :bla
[13] pry(main)> bla(a: "foo")
=> {:foo=>"foo", :bar=>"default"}
```

Reply:
Or are we expecting to always pass it and just give a default when the value is `nil`? 🤔

Reply:
> Or are we expecting to always pass it and just give a default when the value is nil?

Yes, exactly.  Existing sessions won't have a `level_of_authentication`, so they'll pass `nil`.

Though, on reflection, I think it'll be better to handle that case in `decrypt_session`.  I'll push a fixup.